docs/playground/src/PlaygroundConfiguration.ts

```diff
@@ -83,51 +121,61 @@ export const PlaygroundConfiguration: IStringMap<IComponentPlaygroundConfigurati
   },
   Breadcrumb: {
     show: true,
-    element: $$('div', undefined, `<div class="CoveoBreadcrumb"></div><p>Interact with the facet to modify the breadcrumb</p><div class="CoveoFacet" data-field="@objecttype" data-title="Type"></div>`)
+    element: new SectionBuilder()
+      .withComponent('CoveoBreadcrumb')
+      .withDomElement($$('p', {}, 'Interact with the facet to modify the breadcrumb'))
+      .withComponent('CoveoFacet', {
+        'data-field': '@objecttype',
+        'data-title': 'Type'
+      })
+      .build()
   },
   DidYouMean: {
     show: true,
     basicExpression: 'testt',
-    element: $$('div', undefined, `<div class="CoveoDidYouMean"></div><div class='CoveoSearchbox'></div>`)
+    element: new SearchSectionBuilder().withComponent('CoveoDidYouMean').build()
   },
   ErrorReport: {
     show: true,
-    toExecute: ()=> {
+    toExecute: () => {
       Coveo['SearchEndpoint'].endpoints['default'].options.accessToken = 'invalid';
     }
   },
   ExportToExcel: {
     show: true,
-    element: $$('div', undefined, `<div class="CoveoSettings"></div><div class="CoveoSearchbox"></div><div class="CoveoExportToExcel"></div>`),
-    toExecute: ()=> {
-      $$(document.body).find('.CoveoSearchInterface').style.minHeight = '300px';
+    element: new SearchSectionBuilder().withComponent('CoveoExportToExcel').build(),
+    toExecute: () => {
```

Comment:
Possible to remove `{}` brackets and write the function on one line. Same applies to a few other `toExecute()` properties in this file.